### PR TITLE
Issue 29 validation icons

### PR DESCRIPTION
**Pull request by CedricCabessa**
Carm integral, in one PR

### CODE REVIEW

File: gm_pr/static/gm_pr.css [hunk 1]
```diff
@@ -92,3 +92,6 @@ td a:hover {
   text-align: center;
   width: 20px;
 }
+.symbol {
+  font-family: sans-serif;
```
Comment (caarmen) [inline gm_pr/static/gm_pr.css]: Add this attribute to the symbol class:
```
font-weight: normal;
```

Then the icon will appear in Chrome/MacOS.

**Comment by caarmen** [comment]
BTW, I did another PR yesterday which put sans-serif everywhere: https://github.com/CedricCabessa/gm_pr/pull/35

**Comment by CedricCabessa** [comment]
Ok, so if we put all your work together, I have nothing to do :-)
I rebased the branch "css-cleanup" on your "issue 29 validation icons" and everything works

**Comment by caarmen** [comment]
Thanks for combining the branches.

I tested this combined branch and the icons all appeared on firefox/chrome/safari on mac.

**Comment by rchutaux** [comment]
LGTM

**Pull request merged**